python/ctsm/site_and_regional/plumber2_shared.py

```diff
@@ -0,0 +1,25 @@
+"""
+Things shared between plumber2 scripts
+"""
+
+import os
+import pandas as pd
+
+PLUMBER2_SITES_CSV = os.path.realpath(
+    os.path.join(
+        os.path.dirname(__file__),
+        os.pardir,
+        os.pardir,
+        os.pardir,
+        "tools",
+        "site_and_regional",
+        "PLUMBER2_sites.csv",
+    )
```

Comment:
It's cosmetic, but you could use os.path.normpath or os.path.realpath to get a shorter directory name that's easier to read for humans.